app/src/main/java/com/github/se/cyrcle/model/map/MapViewModel.kt

```diff
@@ -24,6 +29,8 @@ class MapViewModel : ViewModel() {
   private val _locationPickerState = MutableStateFlow(LocationPickerState.NONE_SET)
   val locationPickerState: StateFlow<LocationPickerState> = _locationPickerState
 
+  private val _focusMode = MutableStateFlow(false)
+  val focusMode: StateFlow<Boolean> = _focusMode
```

Comment:
I don't think `focusMode` is a good name for a boolean. What does `true` and `false` mean ? 
- The easy solution is to rename it as `isTrackingModeEnable` (or something similar). 
- The more engineered solution (often overkill) is to create an `Enum` class that contains all the possibilities, and you can keep the name

Reply:
I changed to your first proposition , the second one being as you said a bit overkill. Thanks for the propositions !